src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -398,6 +398,10 @@ TEST_F(ConnectionHandlerTest, AddConnection_StopConnection) {
 }
 
 TEST_F(ConnectionHandlerTest, GetConnectionSessionsCount) {
+  EXPECT_TRUE(connection_handler_->getConnectionList().empty());
```

Comment:
@mkorniichuk if it is the only possible default state use ASSERT, if it is precondition for test use ON_CALL, if it is tested functionality - use EXPECT.